client/signup/steps/user/index.jsx

```diff
@@ -149,6 +150,7 @@ export class UserStep extends Component {
 				submitForm={ this.submitForm }
 				submitButtonText={ this.submitButtonText() }
 				suggestedUsername={ this.props.suggestedUsername }
+				isSocialSignupEnabled={ isEnabled( 'signup/social' ) && this.props.flowName === 'social' }
```

Comment:
`UserStep` is a bit large so I'd rather not duplicate it just for our tests (also I will risk making this PR unmergeable because of its size).